samples/subsys/fs/fat_fs/README.rst

```diff
@@ -7,27 +7,64 @@ Overview
 ********
 
 This sample app demonstrates use of the filesystem API and uses the FAT file
-system driver to mount an SDHC card connected over a SPI bus or to an on-chip
-SDHC controller.
+system driver with SDHC card, SoC flash or external flash chip.
 
-Requirements
-************
+To access device the sample uses :ref:`disk_access_api`.
+
+Requirements for SD card support
+********************************
 
 This project requires SD card support and microSD card formatted with FAT filesystem.
 See the :ref:`disk_access_api` documentation for Zephyr implementation details.
+Boards that by default use SD card for storage:
+``arduino_mkrzero``, ``esp_wrover_kit``, ``mimxrt1050_evk``, ``nrf52840_blip``,
```

Comment:
mimrt1050 is in this list twice. I would also add the `mimxrt1060_evk`, `lpcxpresso55s69_cpu0`, `mimxrt1170_evk_cm7`, and `mimxrt1024_evk`. (I believe any board with the `zephyr,sdmmc-disk` compatible in its devicetree should use the SD card by default)

Reply:
Will fix.